### PR TITLE
Support zip files, alternative OS_IMAGE and OS_URI

### PR DESCRIPTION
**Pull request by jedahan**
This let me do:

```bash
export OS_IMAGE=2018-06-27-raspbian-stretch-lite.zip
export OS_URI=http://vx2-downloads.raspberrypi.org/raspbian_lite/images/${OS_IMAGE%%.zip}/${OS_IMAGE}
make run
```
There are other issues with what I am doing (like filetype detection), but this is just to start the conversation.

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -3,6 +3,8 @@ SHELL := /bin/bash
 .DEFAULT_GOAL := build
 
 TAG?=latest
+OS_IMAGE?=ArchLinuxARM-rpi-latest.tar.gz
+OS_URI?=http://archlinuxarm.org/os/$(OS_IMAGE)
```
Comment (jedahan) [inline makefile:7]: These are duplicated with `bin/load-os` not sure the best way to remove duplication.

File: README.md [hunk 2]
```diff
@@ -22,10 +22,20 @@ Ubuntu users should install these packages before building an image:
 apt-get install binfmt-support qemu-user-static make
 ```
 
+### Customization
+
+By exporting `OS_IMAGE` and `OS_URI` you can use a different operating system, like so:
+
+```bash
+export OS_IMAGE=root.tar.xz
+export OS_URI=http://vx2-downloads.raspberrypi.org/raspbian_lite/archive/${OS_IMAGE}
```
Comment (jedahan) [inline README.md]: Raspbian splits up boot and root into two xz files, not sure what th ebest way to deal with them both are

File: makefile (hunk 1 above)
Comment (westlane) [reply makefile:7]: It's smart and helpful to have OS_IMAGE and OS_URI specified in the makefile, and in this case I feel we can keep the duplication as such. My thinking is that some users may wish to run the docker container without this specific makefile, for example from their own build script or as part of a larger toolchain.

**Comment by westlane** [comment]
Thanks for your thoughtful efforts on this, @jedahan ! Very nice addition that helps to expand reach and value for this tool.

**Pull request merged**